.env.demo

```
@@ -230,3 +233,16 @@ PRISMA_LOGS = error
 
 # DB_ALERT_ENABLE=
 # DB_ALERT_EMAILS=
+
+
+
+PROJECT_ID=
+CLIENT_EMAIL=
+PRIVATE_KEY=
+
+CONSUMER_CONFIG_ACK_WAIT=10_000  # IN nanos(10_000)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)